examples/neutron.rs

```diff
@@ -0,0 +1,187 @@
+use localic_utils::{
+    types::contract::{
+        AuctionStrategy, ChainHaltConfig, MinAmount, PairType, PriceFreshnessStrategy,
+    },
+    ConfigChainBuilder, TestContextBuilder,
+};
+use std::error::Error;
+
+/// Demonstrates using localic-utils for neutron.
+fn main() -> Result<(), Box<dyn Error>> {
+    env_logger::init();
+
+    let neutron = ConfigChainBuilder::default()
+        .with_admin_addr("neutron1kuf2kxwuv2p8k3gnpja7mzf05zvep0cyuy7mxg")
```

Comment:
this addr can be a constant and assumed default for all neutron builders. overriding it should be explicit.

Reply:
See comments about default_neutron. This is a one liner now!